app/graphql/client.tsx

```diff
@@ -95,28 +95,46 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
         }`,
       )
 
-      const wsLinkAuth = async () => {
-        if (token) {
-          return {
-            Authorization: getAuthorizationHeader(token),
-          }
-        } else if (appConfig.isAuthenticatedWithDeviceAccount) {
+      const appCheckLink = setContext(async (_, { headers }) => {
+        if (appConfig.isAuthenticatedWithDeviceAccount) {
```

Comment:
why not send the appCheckToken for all levels? this is what I was suggesting yesterday, the only drawback been the bandwidth

Reply:
I had thought that it might incur a cost, but it looks like it is free with a quota of 10,000 calls per day, so I think we can enable it on all calls.